app/views/applications/show.html.erb

```diff
@@ -0,0 +1,12 @@
+<h1><%= @application.name %></h1>
+<h3>Address: <%= @application.address %></h3>
+<h3>Description: <%= @application.description %></h3>
+
+<h3>Requested pet(s):</h3>
+<ul>
+  <% @application.pets.each do |pet| %>
```

Comment:
Let's leave this as is for now, but it may be worth asking about this in office hours/class, I think it might break MVC? I have the gut feeling this would traditionally actually go in the model or controller and then the view would just do `@pets.each`.

It definitely adds another layer for a VERY simple method, so I could be wrong.